apps/src/studio/studio.js

```diff
@@ -2946,7 +2956,7 @@ Studio.callCmd = function (cmd) {
     case 'playSound':
       studioApp.highlight(cmd.id);
       studioApp.playAudio(cmd.opts.soundName, { volume: 1.0 });
-      Studio.playSoundCount++;
+      Studio.trackedBehavior.playSoundCount++;
```

Comment:
Did you intend to change this? I would worry that some existing code (perhaps in a level's `successCondition`) might be depending on the previous name.